lib/providers/hlr-lookups-provider.js

```diff
@@ -44,7 +61,7 @@ HLRLookupsProvider.prototype.hlrLookup = function hlrLookup(number) {
     }
 
     if (!isProviderLookupEnabled) {
-      logger.info(logFunc('Returning dummy lookup result because actual calls are prohibited (env ENABLE_HLR_LOOKUPS not set to true)'));
+      logWarningIfRequired(logger, logFunc);
       if (!dummyResponse) {
         dummyResponse = require('./dummy_data/hlr-lookups-provider.json').data;
```

Comment:
Ok thanks!

My initial through is to use a stub of the lookup service in tests, to fake the answer and use dummy data directly from tests, so the main code does not have condition or test-related logic. Would it be possible?